Source/Scene/PostProcessStageLibrary.js

```diff
@@ -685,6 +695,88 @@ define([
         });
     };
 
+    /**
+     * Creates a post-process stage that applies ACES tonemapping operator.
```

Comment:
What is ACES?  And provide a tad of context please.

Reply:
Same comment for the other tonemapping stages.  Provide enough context so users would know when to select one over the other.  Could make a nice Sandcastle example as well.